src/main/java/se/sundsvall/casedata/integration/emailreader/EmailReaderClient.java

```diff
@@ -19,10 +19,18 @@
 )
 public interface EmailReaderClient {
 
-	@GetMapping("/email")
-	List<Email> getEmail(@RequestParam("municipalityId") final String municipalityId, @RequestParam("namespace") final String namespace);
-
-	@DeleteMapping("/email/{id}")
-	void deleteEmail(@PathVariable String id);
+	@GetMapping("/{municipalityId}/email/{namespace}")
+	List<Email> getEmail(
+		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281")
```

Comment:
Swagger annotations are only useful on the REST-controller, since the clients will not produce any API-documents.